### PR TITLE
Fixed problem with --xplat build when mono does not installed.

### PR DESCRIPTION
**Pull request by dmitriyse**
### What does this implement/fix? Explain your changes.
Fixes --xplat build when no mono available.
...

### Does this close any currently open issues?
https://github.com/pythonnet/pythonnet/issues/571
...

### Any other comments?

...

### Checklist

Check all those that are applicable and complete.

-   [ ] Make sure to include one or more tests for your change
-   [ ] If an enhancement PR, please create docs and at best an example
-   [ ] Add yourself to [`AUTHORS`](../blob/master/AUTHORS.md)
-   [ ] Updated the [`CHANGELOG`](../blob/master/CHANGELOG.md)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -281,7 +281,13 @@ def _get_manifest(self, build_dir):
         return manifest
 
     def _build_monoclr(self):
-        mono_libs = _check_output("pkg-config --libs mono-2", shell=True)
+        try:
```
Comment (Cronan) [inline setup.py]: I think the mono cflags check should be in the try block as well - it also throws in the same way, I believe

Comment (denfromufa) [reply setup.py]: @dmitriyse is this not failing in CI because both Mono and .NET Core are available?

Comment (dmitriyse) [reply setup.py]: Yes, --xplat was working in CI only due to both frameworks installed.

**Comment by Cronan** [comment]
Leave a comment can confirm that I am able to successfully build using this branch

**Comment by denfromufa** [comment]
@Cronan so you can build pythonnet for .NET Core without Mono installed on this Linux machine?

**Comment by Cronan** [comment]
@denfromufa yes, I can build pythonnet for .NET Core without Mono installed on RH Linux Enterprise 7.3 (using the code in this PR)

**Comment by denfromufa** [comment]
ok, I'm merging this pull request, thanks @dmitriyse @Cronan !

**Pull request merged**